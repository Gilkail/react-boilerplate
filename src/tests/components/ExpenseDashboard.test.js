import React from 'react'
import {shallow} from 'enzyme'
import DashboardPage from '../../components/DashboardPage'

test('Should display the dashboard componenet', ()=>{
    const wrapper = shallow(<DashboardPage />)
    expect(wrapper).toMatchSnapshot()
})