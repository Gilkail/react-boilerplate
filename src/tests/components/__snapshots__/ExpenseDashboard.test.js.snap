// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should display the dashboard componenet 1`] = `
<div>
  Dashboard Page Content
</div>
`;
